src/librbd/internal.cc

```diff
@@ -275,7 +275,7 @@ int validate_pool(IoCtx &io_ctx, CephContext *cct) {
     return io_ctx.tmap_update(RBD_DIRECTORY, cmdbl);
   }
 
-  typedef boost::variant<std::string,uint64_t> image_option_value_t;
+  typedef std::variant<std::string,uint64_t> image_option_value_t;
```

Comment:
Nit: Since this line is touched anyway, let's fix the formatting:
```suggestion
  typedef std::variant<std::string, uint64_t> image_option_value_t;
```